.github/workflows/build.yml

```
@@ -18,19 +18,11 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 7.3]
+                php: [7.4]
                 node: [10.x]
                 mysql: [5.7, 8.0]
-                symfony: [^4.4, ^5.2]
-                sylius: [~1.8.0, ~1.9.0]
-
-                exclude:
-                    - # PHP 7.3 does not support "caching_sha2_password" authentication plugin which is a default one in MySQL 8.0
-                        php: 7.3
-                        mysql: 8.0
-                    -
-                        sylius: ~1.8.0
-                        symfony: ^5.2
+                symfony: [^4.4, ^5.2, ^5.3]
```

I guess that there will be the same version of Symfony installed for `^5.2` and `^5.3`
```suggestion
                symfony: [^4.4, ~5.2.0, ~5.3.0]
```